### DISCUSSION TITLE
massCode need you help 🙏

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#61] [Announcements]
Hi there,

I'm not good at PR and creating media noise around anything, I'm just a developer who focuses on code. If you like the project and it is useful to you, a huge request to help get the information to the public, it can be a tweet, article-review, etc. Give a star on GitHub is also a great contribution, because it's a rating and the more stars the project has, the more it will be noticed. 

For my part, I keep trying to post where possible, but I really need your help 🙏

Thanks in advance!

**Comment by v-c0de** [comment]
What you have written is great! And I have been sharing your latest version with colleagues to get them to try it out for the first time
